app/assets/javascripts/ang/controllers/observation_search.js

```diff
@@ -1196,33 +1271,36 @@ function( ObservationsFactory, PlacesFactory, TaxaFactory, shared, $scope, $root
   $scope.preInitialize( );
 }]);
 
+var boundaryBoxStyle = {
```

Comment:
Maybe add a comment here explaining that Google requires this style info be added in code, assuming that's true.